TFE - update docker on RHEL info (#2341)
hashicorp/terraform-website: content/enterprise/requirements/os-specific/rhel-requirements.mdx

### Docker was accidentally upgraded, how can I downgrade?

> The `yum downgrade` command can be used to downgrade the version of Docker that is installed, if it was installed from the EPEL repository.

Reviewer: ```suggestion
You can use the `yum downgrade` command to downgrade the version of Docker that is installed, if it was installed from the EPEL repository.
```

Reviewer: If it wasn't installed this way, are they out of luck?
